segmentation/include/pcl/segmentation/impl/min_cut_segmentation.hpp

```diff
@@ -48,23 +48,13 @@
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 template <typename PointT>
 pcl::MinCutSegmentation<PointT>::MinCutSegmentation () :
-  inverse_sigma_ (16.0),
-  binary_potentials_are_valid_ (false),
-  epsilon_ (0.0001),
-  radius_ (16.0),
-  unary_potentials_are_valid_ (false),
-  source_weight_ (0.8),
   search_ (),
-  number_of_neighbours_ (14),
-  graph_is_valid_ (false),
   foreground_points_ (0),
```

Comment:
These four are vectors as well - so could be removed from here?

search_() - creates an empty shared vector also?